packages/services/api/src/modules/auth/module.graphql.ts

```diff
@@ -53,17 +53,17 @@ export default gql`
     total: Int!
   }
 
-  enum AuthProvider {
-    GOOGLE
-    GITHUB
+  enum AuthProviderType {
+    GOOGLE @tag(name: "public")
```

Comment:
I couldnt quickly find docs on this... but could you add the tag to the enum type to apply it to all its values? I don't see a world where we'd want to only expose some of these values.

Reply:
Our implementation of contract filtering does not support it - I think it should though.